packages/rx_bloc_cli/mason_templates/bricks/lib_auth_apple/__brick__/lib/lib_auth_apple/models/apple_auth_request_model.dart

```diff
@@ -0,0 +1,41 @@
+{{> licence.dart }}
+
+import 'package:json_annotation/json_annotation.dart';
+
+import 'apple_credentials_model.dart';
+
+part 'apple_auth_request_model.g.dart';
+
+@JsonSerializable(includeIfNull: false)
+class AppleAuthRequestModel {
+  AppleAuthRequestModel({
+    this.authorizationCode,
+    this.userIdentifier,
+    this.firstName,
+    this.lastName,
+    this.email,
+    this.identityToken,
+  });
+
+  final String? authorizationCode;
```

Comment:
Are we sure all of the props are nullable ?

Reply:
Not actually, the authorization code won't be null.